docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -1,387 +1,353 @@
+{{% tab name="Linux (Ubuntu)" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+sudo apt-get install git wget flex bison gperf python3 python3-pip python3-venv cmake ninja-build ccache libffi-dev libssl-dev dfu-util libusb-1.0-0
+```
+
+{{% /tab %}}
+{{% tab name="macOS" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+brew install cmake ninja dfu-util
+```
+
+{{% /tab %}}
+{{% /tabs %}}
+
+#### Install and Activate the ESP-IDF
+
+Clone Viam's fork of the ESP-IDF, the development framework for Espressif SoCs (System-on-Chips) supported on Windows, Linux and macOS:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+mkdir -p ~/esp
+cd ~/esp
+git clone --depth 1 -b v4.4.4 --single-branch --recurse-submodules --shallow-submodules https://github.com/npmenard/esp-idf
+```
+
+Then, install the required tools for ESP-IDF:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+cd ~/esp/esp-idf
+./install.sh esp32
+```
+
+To activate ESP-IDF, run the following command to source (`.`) the activation script `export.sh`:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+. $HOME/esp/esp-idf/export.sh
+```
+
+To avoid conflicts with other toolchains, adding this command to your `.bashrc` or `.zshrc` is not recommended.
+Save this command to run in any future terminal session where you need to activate the ESP-IDF development framework.
+
+#### Install the Rust ESP Toolchain and Activate the ESP-RS Virtual Environment
+
+First, install the following tools with `cargo`:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+cargo install espup
+```
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+cargo install cargo-espflash v2.0.0-rc.1
+```
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+cargo install ldproxy
+```
+
+To download and install the esp-rs toolchain, run:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+espup install -s -f ~/esp/export-rs.sh -v 1.67.0
+```
+
+To activate the ESP Rust toolchain, run the following command to source (`.`) the activation script `export-rs.sh`:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+. $HOME/esp/esp-idf/export-rs.sh
+```
+
+To avoid conflicts with other toolchains, adding this command to your `.bashrc` or `.zshrc` is not recommended.
+Save this command to run in any future terminal session where you need to activate the ESP Rust development framework.
+
+## Install the Micro-RDK
+
+### Create a New Robot
+
+Navigate to [the Viam app](https://app.viam.com) and [add a new robot](/manage/fleet/robots/#add-a-new-robot) in your desired location.
+
+* Click on the name of the robot to go to the robot's page.
+* Skip the instructions in the [**Setup** tab](/manage/fleet/robots/#setup), as the setup instructions there are not for microcontrollers.
+* Keep your `Mode` and `Architecture` selections at default.
```

Comment:
This is not needed if we're telling them to skip the Setup page.
Nit (feel free to ignore me): On the 2nd bullet point, I think it's better to tell people to proceed to the Config tab rather than telling them to skip Setup.

Reply:
Done!